### DISCUSSION TITLE
Συμμετοχικο Β1 εντοπισμός κατάλληλης μελέτης περίπτωσης

### DISCUSSION THREAD

**Discussion by AlexandrosV20** [courses-ionio/hci#1665] [Announcements]
Θα ήθελα να ρωτήσω αν το case study μπορει να έχει έμμεση σύνδεση με την διάδραση. Έιδα σε ένα md του case study το xerox parc lab. Μπόρούμε και εμεις να βάλουμε κάτι παρόμοιο ;

**Comment by epidrome** [comment]
θεωρείς πως το ζιροξ παρκ, όπου δημιουργήθηκε το αλτο-σμαλτολκ καθώς και η προσομοίωση (ως φάρσα του γραφείου) σταρ, τα οποία (κακό) αντέγραψε ο στέφανος στο (παραμορφωμένο) μάκιντοσ έχει έμμεση σχέση με την διάδραση?!

σίγουρα είναι δύσκολο να εντοπίσουμε οργανισμούς με παρόμοια επίδραση στον χώρο της διάδρασης όσο το ζιροξ παρκ, επίσης είναι πολύ δύσκολο να εντοπίσετε σε έναν μεγάλο-ευρύ οργανισμό την επίδραση που είχε στην διάδραση ένα εργαστήριο του

οπότε ίσως αντι για οργανισμό για εσάς θα ήταν μάλλον πιο εύκολο να εντοπίσετε κάποιο προϊόν ή λειτουργία του, π.χ., unix, www, wikipedia, etc

**Reply by AlexandrosV20** [reply]
Με τον όρο έμμεση εννοουσα ακριβώς αυτό που είπατε αν μπορούμε να επιλέξουμε κάτι εκτός απο κάποιο προϊόν ή λειτουργία, όπως έναν οργανισμο.

**Reply by p20sami** [reply]
@epidrome Επί αυτού, μπορούμε να διαλέξουμε κάτι το οποίο προέρχεται απο συσκευές διάδρασης? Όπως για παράδειγμα το chiptuning που ουσιαστικά ήταν η παραγωγή μουσικής απο εξαρτήματα πολυμεσικών/διαδραστικών συστημάτων?

**Reply by epidrome** [reply]
@p20sami παράδειγμα μουσικής τεχνολογίας με ιδιαίτερη διάδραση:
https://pibook.epidro.me/case-study/reactable/
